docs/user-guides/mirring-configuration.md

```diff
@@ -0,0 +1,95 @@
+# Mirror Configuration
+
+This page describes how to enable mirroring features on the Vald cluster.
+
+Before using the mirroring functions, please check [the Vald Mirror Gateway document](https://vald.vdaas.org/docs/overview/component/mirror-gateway) for what you can do.
+
+## Requirement
+
+- Vald version: v1.8
+- The number of Vald clusters: 2~
+
+## Configuration
+
+This section shows how to configure values.yaml to enable Vald Mirror Gateway and how to interconnect Vald Mirror Gateways.
+
+The setting points are the followings:
+
+1. Enable the Vald Mirror Gateway using Helm values configuration
+2. Interconnect Vald Mirror Gateways using custom resource configuration
+
+### Helm Values Configuration
+
+The Helm values configuration is required for each Vald cluster to be deployed.
+
+It is easy to enable the mirroring feature.
+
+```yaml
+---
+gateway:
+  mirror:
+    enabled: true
+```
+
+If you want to make more detailed settings, please set the following parameters.
+
+```yaml
+gateway:
+  mirror:
+  ...
+    gateway_config:
+    ...
+      # gRPC client configuration (overrides defaults.grpc.client)
+      client: {}
+      # The interval to advertise addresses of Mirror Gateway to other Mirror Gateway.
+      advertise_interval: "1s"
+      # The target namespace to discover ValdMirrorTarget (CR) resource.
+      # The default value is its own namespace.
+      namespace: "vald"
+      # The group name of the Mirror Gateways (optional).
+      # It is used to discover ValdMirrorTarget resources (CR) with the same group name.
+      # The default value is empty.
+      group: "group1"
+      # The duration to discover other mirror gateways in the same group.
+      discovery_duration: 1s
+      # The colocation name of the data center (optional).
+      colocation: "dc1"
+```
+
+Configuring the cluster role setting is required when you deploy Vald clusters with Vald Mirror Gateway on multiple namespaces in the Kubernetes cluster.
+
+Please refer to [Cluster Role Configuration](https://vald.vdaas.org/docs/user-guides/cluster-role-binding/) about cluster role settings for Mirror Gateway.
```

Comment:
Did you add the cluster role configuration about the mirror GW?

Reply:
Sorry, I forgot... 🙏 
So I just added that document, could you please check it?
https://github.com/vdaas/vald/pull/2105/commits/b424727085e5c0b19a5b3ad5febc26c837c00f7a